lib/Devel/Probe.pm

```diff
@@ -47,8 +43,10 @@ sub config {
             my $file = $action->{file};
             next unless $file;
 
-            my $type_name = $action->{type} // 'once';
-            my $type = $probe_type_name_to_type{$type_name} // 1;
+            my $type = $action->{type} // ONCE;
+            if ($type ne ONCE && $type ne PERMANENT) {
```

Comment:
These might as well be `!=`, right? Or do you want to prevent against someone setting type to `CRAP`?

Reply:
Yeah, exactly: I thought it was a better error message to see just the `croak`, rather than a  `Argument "CRAP" isn't numeric in numeric eq (==)` plus the `croak`.